benefits/core/migrations/0002_data.py

```diff
@@ -13,6 +13,11 @@ def load_data(app, *args, **kwargs):
     mst_senior_type = EligibilityType.objects.create(
         name="senior", label="Senior Discount (MST)", group_id=os.environ.get("MST_SENIOR_GROUP_ID", "group1")
     )
+    mst_veteran_type = EligibilityType.objects.create(
+        name="veteran",
+        label="Veteran Discount (MST)",
+        group_id=os.environ.get("MST_VETERAN_GROUP_ID", "group3"),
+    )
```

Comment:
@angela-tran @thekaveman Is this all we need for #1426? Does this PR also close #1426 or is there something more that it entails? Wasn't sure.

Reply:
I think #1426 also includes creating the group in Littlepay and adding the group ID to Key Vault in Azure for dev and test.

Reply:
This change seems out of scope for this PR? Why not leave it all for the PR that closes #1426?

Reply:
@machikoyasuda it looks like you did fold #1426 into this one (creating the LP group, configuring it in KeyVault, linking to the EligibilityType)?

Reply:
I just wasn't really sure what to put here otherwise: https://github.com/cal-itp/benefits/pull/1436/files#r1238987028

Reply:
https://github.com/cal-itp/benefits/pull/1436#discussion_r1238989101